CHANGELOG.md

```diff
@@ -1,19 +1,30 @@
 # Changelog
 
+## [Version 0.3.0 (Public Preview)](https://github.com/awslabs/aws-mysql-jdbc/releases/tag/0.3.0) - 2021-11-18
+
+### Added
+* AWS IAM Authentication is now supported. Usage instructions can be found [here](https://github.com/awslabs/aws-mysql-jdbc#aws-iam-database-authentication).
+
+### Improvements
+* Java 8 is now enforced in the build process.
+* Resolved an issue for when connecting with an invalid connection after a valid connection. Before this would allow the user to connect despite having invalid information, such as a username.
```

Comment:
The second sentence looks a bit weird, what do you think of:
Users were able to connect with cached information after a valid connection despite providing invalid information.